src/views/Layout.jsx

```diff
@@ -15,22 +17,27 @@ export function Layout() {
 		<>
 			<div className="Layout">
 				<header className="Layout-header">
-					<h1>Smart shopping list</h1>
+					{/* <h1>Smart shopping list</h1> */}
```

Comment:
**Chore:** Another comment we can get rid of! The ones starting on line 7 are also no longer relevant